packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/catalog/services.py

```diff
@@ -191,3 +196,42 @@ async def _call(
         service_key=service_key,
         service_version=service_version,
     )
+
+
+@log_decorator(_logger, level=logging.DEBUG)
+async def batch_get_my_services(
+    rpc_client: RabbitMQRPCClient,
+    *,
+    product_name: ProductName,
+    user_id: UserID,
+    ids: list[
+        tuple[
+            ServiceKey,
```

Comment:
I think we used to have an object that contains the service key and version. Actually we still do: packages/models-library/src/models_library/services_base.py
why not using this instead of tuple?

Reply:
I see your point but 

- I would also argue then that all functions with separated input parameters
```
    service_key: ServiceKey,
    service_version: ServiceVersion,
```
should also be replaced by `ServiceKeyVersion` model?
- i am connecting with other functions that already return `list[tuple[ServiceKey, ServiceVersion]]`
+/- `ServiceKeyVersion`  will validate again